html/controls/plt/plt-input.js

```diff
@@ -442,17 +455,28 @@ function preparePenaltyEditor() {
 
 	function initialize() {
 		
-		var topTable = $('<table width="80%">').appendTo($('#PenaltyEditor'));
+		var topTable = $('<table width="100%">').appendTo($('#PenaltyEditor'));
 		var tr = $('<tr>').appendTo(topTable);
-		$('<td width="33%">').append($('<span>').text('Period: ')).append($('<button>').addClass('period_minus').text('-1'))
-			.append($('<select>').addClass('Period')).append($('<button>').addClass('period_plus').text('+1')).appendTo(tr);
-		$('<td width="33%">').append($('<span>').text('Jam: ')).append($('<button>').addClass('jam_minus').text('-1'))
-			.append($('<select>').addClass('Jam')).append($('<button>').addClass('jam_plus').text('+1')).appendTo(tr);
-		$('<td width="33%">').append($('<button>').addClass('clear').text('Clear')).appendTo(tr);
+		$('<td width="35%">').append($('<span>').text('Period: ')).append($('<button>').addClass('period_minus').text('-1').button())
+			.append($('<select>').addClass('Period')).append($('<button>').addClass('period_plus').text('+1').button()).appendTo(tr);
+		$('<td width="35%">').append($('<span>').text('Jam: ')).append($('<button>').addClass('jam_minus').text('-1').button())
+			.append($('<select>').addClass('Jam')).append($('<button>').addClass('jam_plus').text('+1').button()).appendTo(tr);
+		$('<td width="15%">').append($('<button>').addClass('set').text('Set Jam').button()).appendTo(tr);
```

Comment:
This shouldn't be required, changes should be submitted immediately.

Reply:
Since the only way to submit changes to a penalty is via their position in the ordered list and changing the jam number may change that position, changing the jam immediately could result in subsequent changes being submitted for the wrong penalty unless we close the window. But closing the window on every +1/-1 would be incredibly annoying.

Reply:
Then the set jam shouldn't show when initially creating a penalty. And it should be jam/period.

Reply:
Can the Set Jam button be grouped more with the jam/period? As-is isn't not obvious that they're related. Maybe have Served/Clear on one row? For consistency, Clear should probably be Delete too.